src/kibana/components/stringify/types/String.js

```diff
@@ -36,20 +36,31 @@ define(function (require) {
       { id: false, name: '- none -' },
       { id: 'lower', name: 'Lower Case' },
       { id: 'upper', name: 'Upper Case' },
-      { id: 'short', name: 'Short Dots' }
+      { id: 'short', name: 'Short Dots' },
+      { id: 'binary', name: 'Base64 Decode'}
```

Comment:
Since it looks like `id` is an arbitrary name we're providing, can you change it to base64 instead of binary? The source data is both not binary and definitely base-64, by definition ;)